test/authenticateUsernamePasswordForToken.js

```diff
@@ -176,7 +176,7 @@ describe('authenticateUsernamePasswordForToken',function() {
         request(app)
           .post(tokenEndpoint)
           .send(mockLoginPost)
-          .expect('set-cookie', xsrfTokenCookieExpr, done)
+          .expect('set-cookie', xsrfTokenCookieExpr)
```

Comment:
I couldn't get the tests to run successfully on master without removing the call to done here, since it's called on the next expect line.  It would error out saying that done was called more than once.